app/assets/styles/vendor/map.css

```diff
@@ -69,4 +66,17 @@ table {
   font-family: Roboto;
   font-size: 13px;
   font-weight: 300;
-} 
\ No newline at end of file
+}
+
+#sidebar {
```

Comment:
one of these two #sidebar-s is redundant? Also, some of the values are conflicting, make sure to leave the right ones :)